prettier.config.js

```diff
@@ -2,4 +2,12 @@ module.exports = {
   printWidth: 100,
   singleQuote: true,
   trailingComma: 'all',
+  overrides: [
```

Comment:
I feared that we needed separate runners. I'm glad this is already built-in, thanks!

Could you add a comment explaining we need this for typescript 3.2?